backend/user-service/src/types/EmailVerificationDto.ts

```diff
@@ -0,0 +1,31 @@
+import { IsDate, IsEmail, IsNotEmpty, IsString, ValidationError, validate } from 'class-validator'
+
+import { TypedRequest } from './TypedRequest'
+
+export class EmailVerificationDto {
+    @IsEmail()
+    @IsNotEmpty()
+    email: string
+
+    @IsString()
+    verificationToken?: string
```

Comment:
the token should be a number yes? you can use the @IsNumericString validator

Reply:
yep, but the default value was an empty string, now the updated default value is '0'.